ChangeLog.d/issue4274.txt

```diff
@@ -0,0 +1,4 @@
+Changes
+   * Create 4 byte reading macros in relevant files in library/:
```

Comment:
Strictly speaking, 4 should probably be written as the word "four" here.

Reply:
Are these four macros (four byte-reading macros), or macros that read four bytes (four-byte reading macros)? But anyway, this is not a user-visible change, so it should not have a changelog entry.

Reply:
I was just removing it. I must've forgotten as I did it to the other 2 PRs